tests/core/test_simd_float32x4.c

```diff
@@ -5,9 +5,25 @@
 #include <math.h>
 #include <assert.h>
 
+void printFloat(float n) {
```

Comment:
I presume the print format changed here somehow, and `printf("%f", nan);` no longer prints the string "nan"? What does it print now?